src/core/util/util.js

```diff
@@ -204,44 +204,56 @@ function UtilFactory($document, $timeout, $compile, $rootScope, $$mdAnimate, $in
      *     use the passed parent element.
      */
     disableScrollAround: function(element, parent, options) {
+      options = options || {};
+
       $mdUtil.disableScrollAround._count = $mdUtil.disableScrollAround._count || 0;
```

Comment:
So after checking, this does not work, because `count` is sometimes undefined. 

```js
Math.max(0, $mdUtil.disableScrollAround._count || 0);
```

Doing this ensures, that the count is not smaller than zero.

> Ternary is unnecessary here, duplicating the count three times is too much.